README.md

```diff
@@ -12,96 +12,96 @@ The OpenTelemetry Operator is an implementation of a [Kubernetes Operator](https
 It helps deploy and manage [Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
 
 ## Getting started
+### 1. Deploy the [cert-manager](https://cert-manager.io/docs/)
```

Comment:
Most clusters should probably have this so we should may be mention this is a conditional step.